pipelines/tests/unit/utils/test_sql.py

```diff
@@ -33,13 +33,17 @@ def setup(self, mock_template_dir):
         write_template_to_file(mock_template_dir / INVALID_FILE_NAME, INVALID_TEMPLATE)
 
     @patch("builtins.print")
-    def test__validate_and_print_result_returns_correct_passing_result(self, mock_print, mock_template_dir):
```

Comment:
Any changes to this file just come from running [`black .`](https://pypi.org/project/black/).